caravel/controllers/listings.py

```diff
@@ -105,4 +105,10 @@ def create(keyword):
 @app.route('/logout')
 def logout():
     session.clear()
+    return redirect(url_for('index'))
+
+@app.route('/test')
+def test():
```

Comment:
Er, what? Remember, app versions all use the same shared key — so this gives someone admin access on all versions of the app!